packages/app/cypress/e2e/reporter_header.cy.ts

```diff
@@ -0,0 +1,84 @@
+describe('Reporter Header', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('cypress-in-cypress')
+    cy.openProject('cypress-in-cypress')
+    cy.startAppServer()
+    cy.visitApp()
+    cy.contains('dom-content.spec').click()
+  })
+
+  context('Specs Shortcut', () => {
+    it('displays the currently running spec as highlighted in the Specs List', () => {
+      cy.location().should((location) => {
+        expect(location.hash).to.contain('dom-content.spec')
+      })
+
+      cy.get('[data-selected-spec="true"').should('contain', 'dom-content')
+      cy.get('[data-selected-spec="false"').should('have.length', '1')
+    })
+
+    it('will display the extension in the list as a different highlighted color than the filename', () => {
```

Comment:
This test seems to suggest the filename + extesion are one color, not different.

Reply:
I also removed this from the E2E section of the spreadsheet since it is better covered in CT test (percy snapshot)